shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/antlr4/imports/opengauss/BaseRule.g4

```diff
@@ -1276,6 +1276,18 @@ aliasClause
     | colId
     ;
 
+directoryname
+    : name
+    ;
+
+pathString
+    : stringLiterals
+    ;
+
+stringLiterals
```

Comment:
Can we remove this rule? And then modify `pathString` to `STRING_`.

Reply:
sure.